src/aws-metadata.js

```diff
@@ -0,0 +1,81 @@
+import request from 'request';
+import async from 'async';
+import {Logger} from './Logger';
+
+/*
+  Utility class for pulling AWS metadata that Eureka requires when registering as an Amazon instance (datacenter).
+*/
+export class AwsMetadata {
```

Comment:
Since usage of this class is encapsulated in the client... I'm going to say this doesn't matter anyway.